action.yml

```diff
@@ -1,22 +1,29 @@
-# action.yml
 name: 'Deploy to Aptible'
-description: 'Deploy an app from a Dockerfile to Aptible'
+description: 'Deploy an App to Aptible'
 inputs:
   username:
     description: 'Aptible username'
     required: True
   password:
     description: 'Aptible password'
     required: True
-  environment: 
-    description: 'Aptible environment'
+  environment:
+    description: 'Aptible environment handle'
     required: True
-  app: 
+  app:
     description: 'App handle'
     required: True
-  docker_img: 
-    description: 'Docker image'
+  type:
+    description: 'Deploy strategy "git" or "deploy"'
     required: True
+    default: 'git'
+  git_remote:
+    description: 'Aptible git remote domain'
+    required: False
+    default: elb-aptible-us-east-1-81550.aptible.in
```

Comment:
That's my bad. Eric asked me what the hostname for primetime was and I didn't see a user domain on it so I just gave him the internal hostname. I overlooked the possibility of using a custom cert which would cause the endpoint to not have user domain. We should use `git.aptible.com`.

Reply:
primetime.aptible.com ?

Reply:
Or no certificate because it's using ssh not http 🤦

Reply:
> primetime.aptible.com ?

Is that what we should use for a git push?

Reply:
> We should use primetime.aptible.com, since it's a different service than Megatron (git.aptible.com).

Is that the right service to push to? Normally a user would push to `beta.aptible.com` or `git.aptible.com` which both point to megatron, right?